inc/modules/export/epub/templates/epub3/opf.php

```diff
@@ -69,9 +71,17 @@
 		if ( ! empty( $meta['pb_author_file_as'] ) ) {
 			echo $meta['pb_author_file_as'];
 		} elseif ( ! empty( $meta['pb_author'] ) ) {
-			echo $meta['pb_author'];
+			$nameparser = new Parser();
+			try	{
+				$author = $nameparser->parse( $meta['pb_author'] );
+				echo $author->getLastName() . ', ' . $author->getFirstName();
+			} catch ( Exception $e ) {
```

Comment:
Should be `\Exception`. Optionally, my preference would be to catch `\HumanNameParser\Exception\NameParsingException` instead of searching the string for the error.